docker/services/api_compat/api_compat.finish

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
this file needs to be duplicated for each service. we should have an abstracted function that contains all duplicate code, and call that from each service's `finish` file.

Reply:
You create a generic log function that is included by all your finish files:

```shell
function log {
        printf "%(%Y-%m-%d %H:%M:%S %Z)T ${LOGTAG:-$0} "
        echo $@
}
```

Then at the top of each file:

```shell
export LOGTAG="api_compat"
. common.sh
```

Reply:
A generic log function is a good idea:

```shell
function log {
        printf "%(%Y-%m-%d %H:%M:%S %Z)T ${LOGTAG:-$0} "
        echo $@
}
```

Then at the top of each finish file:

```shell
export LOGTAG="api_compat"

. /etc/functions.sh
```